responses/en/HassGetState.yaml

```diff
@@ -0,0 +1,38 @@
+language: en
+responses:
+  intents:
+    HassGetState:
+      one: "{{ slots.name }} is {{ state.state_with_unit }}"
+      one_yesno: |
+        {% if query.matched: %}
+        Yes
+        {% else: %}
+        No, {{ slots.name }} is {{ state.state_with_unit }}
+        {% endif %}
+      any: |
+        {% if query.matched: %}
+        Yes: {{ query.matched | map(attribute="name") | sort | join(", ") }}
+        {% else: %}
+        No
+        {% endif %}
+
+      all: |
+        {% if not query.unmatched: %}
+        Yes
+        {% else: %}
+        No: {{ query.unmatched | map(attribute="name") | sort | join(", ") }}
+        {% endif %}
+
+      which: |
+        {% if not query.matched: %}
+        Not any
+        {% else: %}
+        {{ query.matched | map(attribute="name") | sort | join(", ") }}
+        {% endif %}
+
+      how_many: |
+        {% if not query.matched: %}
```

Comment:
I guess this is not necessary?

Reply:
Without this, it's an empty box in Assist :/